content/en/llm_observability/setup/sdk/_index.md

```diff
@@ -356,17 +356,29 @@ def get_relevant_docs(question):
 
 ## Tracking user sessions
 
-Session tracking allows you to associate multiple interactions with a given user. When starting a root span for a new trace or span in a new process, specify the `session_id` argument with the string ID of the underlying user session:
+Session tracking allows you to associate multiple interactions with a given user. When starting a root span for a new trace or span in a new process, specify the `session_id` argument with the string ID of the underlying user session, which will be submitted as a tag on the span. Optionally, you can also specify the `user_handle`, `user_name`, and `user_id` tags.
 
 {{< code-block lang="python" >}}
 from ddtrace.llmobs.decorators import workflow
 
 @workflow(session_id="<SESSION_ID>")
-def process_message():
-    ... # user application logic
+def process_user_message():
+    LLMObs.annotate(
+        ...
+        tags = {"user_handle": "poodle@dog.com", "user_id": "1234", "user_name": "poodle"}
+    )
     return 
 {{< /code-block >}}
 
+### Session Tracking Tags
+
+| Tag | Description |
+|---|---|
+| `session_id` | ID representing a single user session, e.g. a chat session |
+| `user_handle` | The handle for the user of the chat session |
```

Comment:
I added support for both `user_handle` and `user_name` since they are both datadog concepts but honestly they are repetitive -- we really need to think through this data and also how it ties into RUM sessions. But for now this is good